stats/thresholds.go

```diff
@@ -81,26 +83,62 @@ func (t *Threshold) Run() (bool, error) {
 	return b, err
 }
 
+type ThresholdConfig struct {
+	Threshold    string `json:"threshold"`
+	AbortOnTaint bool   `json:"abortOnTaint"`
```

Comment:
When reading the code now I think `abortOnFail`, that you've used in other places, sounds better than `abortOnTaint` (and will probably be easier to understand on first look for people reading script code), so I'm of the opinion that we should change the user facing parameter to be called `abortOnFail`.

Reply:
Totally agree, changed all `abortOnTaint` references to `abortOnFail` for consistency.